src/common/logger/logger_settings.h

```diff
@@ -13,6 +13,7 @@ struct LogSettings
     inline const static std::wstring actionRunnerLogPath = L"RunnerLogs\\action-runner-log.txt";
     inline const static std::string launcherLoggerName = "launcher";
     inline const static std::wstring launcherLogPath = L"LogsModuleInterface\\launcher-log.txt";
+    inline const static std::wstring espressoLogPath = L"Logs\\pt-runner-log.txt";
```

Comment:
Why is it called `pt-runner`?

Reply:
@enricogior because this log file is only generated when using the PT runner hook. The application itself has another log, that is being written to the module log folder.

Reply:
The fact that the content of this log file is related to the PT Runner hook has nothing to do with the naming of the logs that needs to be consistent with the fact that this is a logfile generated by the Espresso module and not by the PT Runner.

Reply:
It's inside the Espresso folder, though? So from the hierarchy, wouldn't it be clear that this is related to Espresso?

Reply:
It doesn't matter where it is, each log should have a consistent naming and should not use the name reserved for other components.

Reply:
@enricogior covered in #11341

Reply:
Thanks!